lib/src/score_emitter.dart

```diff
@@ -3,14 +3,16 @@
 // BSD-style license that can be found in the LICENSE file.
 
 abstract class ScoreEmitter {
-  void emit(String testName, double value);
+  void emit(String testName, double value,
+      {String metric = 'RunTime', String unit});
```

Comment:
It's good to note that this is a breaking change to anyone who `implements ScoreEmitter`.

https://github.com/search?q=%22implements+ScoreEmitter%22&type=code

Is the breaking change worth it?

Reply:
It is worth it, but it should be made in a major release of benchmark_harness. I'm reverting this interface change, using a new final subclass instead, and filing an issue to make the change in a major release.

Reply:
Thank you